index.html

```diff
@@ -0,0 +1,233 @@
+    </header>
+    <main>
+        <section class="container">
+
+            <div class="text">
+                <h1> Hey There. I'm  Annah</h1>
+
+
+                <h3>I'm a Software Developer.</h3>
+
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+
+            </div>
+            <div class="icon">
+                <ul>
+                    <li><a href="#"><img src="./images/img1.png" alt="Github-icon"></a></li>
+                    <li><a href="#"><img src="./images/img2.png" alt="Linkedin-icon"></a></li>
+                    <li><a href="#"><img src="./images/img3.png" alt="Twitter-icon"></a></li>
+                    <li><a href="#"><img src="./images/img4.png" alt="AngelList-icon"></a></li>
+                    <li><a href="#"><img src="./images/img5.png" alt="Medium-icon"></a></li>
+
+                </ul>
+            </div>
+
+
+
+        </section>
+    
+        
+    
+        
+        <section  class= "work-container">
+            <h1 class="title"> My Recent Works</h1>
+            <div class="feature-project">
+                
+                <div class="phimg">
+                    <img src="./images/placeholder.png" alt="placeholder-img">
+                </div>
+                    <h2>Multi-Post Stories</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                         industry's standard dummy text ever since the 1500s, when an unknown printer took a 
+                         standard dummy text.
+                    </p>
+    
+                <div class="languages">
+                    <ul>
+                        <li>css</li>
+                        <li> html</li>
+                        <li class="bootstrap">bootstrap</li>
+                        <li>Ruby</li>
+                       </ul>
+                </div>
+
+                <a href="#">See Project</a>
+            </div>
+
+                <div class="projects">
+                    <div class="project">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads
+                             no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="languages">
+                            
+                            <ul>
+                                <li class="project-language"> Html</li>
+                                <li class="project-language"> Bootstrap</li>
+                                <li class="project-language"> Ruby</li>
+                            </ul>
+                        </div>
+                        
+                            <a href="#" class="project-btn">See project</a>
```

Comment:
- [ ] As defined in the Figma template your `See Project` buttons/links should have stylings for `:active` and `:hover`

![btns-styles](https://user-images.githubusercontent.com/36542986/134499882-414b9510-aad2-4eb6-b612-8cecfe1f3f98.jpg)